packages/contracts/contracts/compound/CTokenFirstExtension.sol

```diff
@@ -657,6 +638,35 @@ contract CTokenFirstExtension is
     return balance;
   }
 
+  function swapCollateral() public {
+    uint256 underlyingBalance = asCToken().balanceOfUnderlying(msg.sender);
+    asCToken().flash(underlyingBalance, new bytes(0));
+  }
+
+  function receiveFlashLoan(uint256 amount, bytes calldata data) public {
+    (address borrower, address outputToken, address newCollateralMarket, address target, bytes memory swapData) = abi
+      .decode(data, (address, address, address, address, bytes));
```

Comment:
actually, is this fn even used or dead code that is to be removed?

Reply:
Correct, this def needs to be removed.